platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygMentionType.swift

```diff
@@ -32,15 +36,14 @@ public extension WysiwygMentionType {
     }
 }
 
-public extension PatternKey {
-    var mentionType: WysiwygMentionType? {
-        switch self {
-        case .at:
-            return .user
-        case .hash:
-            return .room
-        case .slash:
-            return nil
-        }
```

Comment:
There's a bunch of similar code in web too. I wonder if there would be value in us storing the actual character in the